project/Dependencies.scala

```diff
@@ -124,17 +121,18 @@ object Dependencies {
     * Related Slick PR: https://github.com/slick/slick/pull/2101
   */
   private val slickV = "3.3.2-2076hotfix" // scala-steward:off (CROM-6620)
-  private val snakeyamlV = "1.28"
-  private val specs2MockV = "4.10.6"
+  private val snakeyamlV = "1.29"
+  private val specs2MockV = "4.12.3"
   private val sprayJsonV = "1.3.6"
   private val sttpV = "1.5.19" // scala-steward:off (CROM-6564)
-  private val swaggerParserV = "1.0.54"
+  private val swaggerParserV = "1.0.55"
   private val swaggerUiV = "3.23.11" // scala-steward:off (CROM-6621)
-  private val tikaV = "1.25" // scala-steward:off (CROM-6771) - javax vs jaxb
+  private val testContainersScalaV = "0.39.6"
+  private val tikaV = "2.1.0"
   private val typesafeConfigV = "1.4.1"
-  private val workbenchGoogleV = "0.15-2fc79a3"
-  private val workbenchModelV = "0.14-27810079-SNAP"
-  private val workbenchUtilV = "0.6-27810079-SNAP"
```

Comment:
`-SNAP` ⁉️ 😬

Reply:
Desperate times... (and not waiting for releases) 🤷